src/gluonts/model/forecast_generator.py

```diff
@@ -52,49 +52,26 @@ def predict_to_numpy(prediction_net, args) -> np.ndarray:
     raise NotImplementedError
 
 
-@singledispatch
-def recursively_zip_arrays(x) -> Iterator:
+def unbatch(x) -> Iterator:
     """
     Helper function to recursively zip nested collections of arrays.
```

Comment:
Not sure what the best description here would be; “unbatch” feels self-explicative